### PR TITLE
Misc lang fixes to Admin, as pointed out by @torvista

### PR DESCRIPTION
**Pull request by drbyte**

### CODE REVIEW

File: admin/banner_manager.php [hunk 1]
```diff
@@ -271,9 +271,9 @@ function init()
...
-            <td class="smallText" align="center" width="100"><?php echo TEXT_LEGEND_BANNER_ON_SSL . '<br />' . zen_image(DIR_WS_IMAGES . 'icon_blue_on.gif', IMAGE_ICON_BANNER_ON_SSL_ON) . '&nbsp;' . zen_image(DIR_WS_IMAGES . 'icon_blue_off.gif', IMAGE_ICON_BANNER_ON_SSL_OFF); ?></td>
+            <td class="smallText" align="center" width="100"><?php echo TABLE_HEADING_STATUS . '<br />' . zen_image(DIR_WS_IMAGES . 'icon_green_on.gif', IMAGE_ICON_STATUS_ON) . '&nbsp;' . zen_image(DIR_WS_IMAGES . 'icon_red_on.gif', IMAGE_ICON_STATUS_OFF); ?></td>
             <td class="smallText" align="center" width="100"><?php echo TEXT_LEGEND_BANNER_OPEN_NEW_WINDOWS . '<br />' . zen_image(DIR_WS_IMAGES . 'icon_orange_on.gif', IMAGE_ICON_BANNER_OPEN_NEW_WINDOWS_ON) . '&nbsp;' . zen_image(DIR_WS_IMAGES . 'icon_orange_off.gif', IMAGE_ICON_BANNER_OPEN_NEW_WINDOWS_OFF); ?></td>
+            <td class="smallText" align="center" width="100"><?php echo TEXT_LEGEND_BANNER_ON_SSL . '<br />' . zen_image(DIR_WS_IMAGES . 'icon_blue_on.gif', IMAGE_ICON_BANNER_ON_SSL_ON) . '&nbsp;' . zen_image(DIR_WS_IMAGES . 'icon_blue_off.gif', IMAGE_ICON_BANNER_ON_SSL_OFF); ?></td>
```
Comment (drbyte) [inline admin/banner_manager.php]: FYI: This changes both the text of the legend and also fixes the display-order of legend columns

File: admin/includes/languages/english.php [hunk 2]
```diff
@@ -203,7 +203,7 @@ function zen_date_raw($date, $reverse = false) {
 define('BOX_LOCALIZATION_ORDERS_STATUS', 'Orders Status');
 
 // gift vouchers box text
-define('BOX_HEADING_GV_ADMIN', TEXT_GV_NAME . '/Coupons');
+define('BOX_HEADING_GV_ADMIN', 'Discounts');
```
Comment (drbyte) [inline admin/includes/languages/english.php]: FYI: This shortens the Admin menu width

File: admin/includes/languages/english/whos_online.php [hunk 3]
```diff
@@ -43,9 +43,11 @@
 define('WHOIS_TIMER_INACTIVE', 180); // seconds when considered inactive - 180 default = 3 minutes
 define('WHOIS_TIMER_DEAD', 540); // seconds when considered dead - 540 default = 9 minutes
 define('WHOIS_SHOW_HOST', '1'); // show Last Clicked time and host name - 1 default
-define('WHOIS_REPEAT_LEGEND_BOTTOM', '12'); // show legend on bottom when more than how many entries - 12 default
-define('OFFICE_IP_TO_HOST_ADDRESS', 'OFF');
+define('WHOIS_REPEAT_LEGEND_BOTTOM', '12'); // show legend on bottom when more than how many entries - 12 default
 
+define('TEXT_DUPLICATE_IPS', 'Duplicate IP Addresses: ');
+define('TEXT_TOTAL_UNIQUE_USERS', 'Total Unique Users: ');
```
Comment (drbyte) [inline admin/includes/languages/english/whos_online.php]: FYI: This removes hard-coded text in whos-online

File: admin/includes/menu.css [hunk 4]
```diff
@@ -47,13 +47,13 @@
     background-color: #d7d6cc;
 }
 .navbar-nav li>a.dropdown-toggle {
-    font-size: 1.3em;
+    font-size: 1.2em;
```
Comment (drbyte) [inline admin/includes/menu.css]: FYI: Just shrinks menu text a little. And padding adjusted too.

**Comment by zcwilt** [comment]
:+1:

**Comment by ajeh** [comment]
:+1:

**Pull request merged**